libraries/ae-inventory-processor/src/main/java/org/metaeffekt/core/document/report/DocumentDescriptorReportGenerator.java

```diff
@@ -178,28 +178,29 @@ private static void generateInventoryReports(DocumentDescriptor documentDescript
                         }
                 }
 
-                report.setReferenceInventory(inventoryContext.getReferenceInventory());
+                if (inventoryContext.getReferenceInventoryContext() != null) {
```

Comment:
I do not understand this. Why this if else block? Are componentsPath and licensesPath optional? They should be. Also either or can be specified.